### ISSUE TITLE
Card/Platform Support?

### ISSUE THREAD

**Issue by ghost** [eholk/RustGPU#1]
I imagine this works only on NVidia cards? I'd be delighted if it works on AMD/Other!

**Comment by eholk** [comment]
Correct, this only supports NVIDIA. At the time NVIDIA was the best option because we could use the LLVM backend to compile to PTX. Now that SPIR exists, it might be possible to support more platforms.

**Comment by ghost** [comment]
That would be pretty awesome, if/when it happens. I'll hope for it, I guess! It's not an area I can contribute to at present, sadly. :)
